ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/WizardAction.java

```diff
@@ -779,6 +779,18 @@ public ToolbarButtonWithTooltip browseBtn() {
 			return _newModulesExtWizard.getBrowseBtn();
 		}
 
+		public ToolbarButtonWithTooltip deleteBtn() {
+			return _newModulesExtWizard.getDeleteBtn();
+		}
+
+		public ToolbarButtonWithTooltip getAddOriginMoudleBtn() {
+			return _newModulesExtWizard.getAddOriginMoudleBtn();
+		}
+
+		public Text getOriginalModule() {
+			return _newModulesExtWizard.getOriginalModule();
```

Comment:
try to get the original original module instead of Blank, otherwise, we won't add test code for this